jenkins/ocp-config/build/bc.yml

```diff
@@ -33,10 +33,10 @@ parameters:
   required: true
   description: Base image for Jenkins agent base
 - name: JENKINS_MASTER_DOCKERFILE_PATH
-  value: Dockerfile.centos7
+  value: Dockerfile.rhel7
```

Comment:
Why is this not `ubi8`?

Reply:
We have a rhel dockerfile for jenkins master. But we might as well remove it also and maintain ubi only